react-file-uploads/server/index.js

```diff
@@ -0,0 +1,64 @@
+require('dotenv/config');
+const pg = require('pg');
+const express = require('express');
+const ClientError = require('./client-error');
+const errorMiddleware = require('./error-middleware');
+const uploadsMiddleware = require('./uploads-middleware');
+
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const app = express();
+
+const jsonMiddleware = express.json();
+
+app.use(jsonMiddleware);
+
+app.post('/api/uploads', uploadsMiddleware, (req, res, next) => {
+  const { caption } = req.body;
+  if (!caption) {
+    throw new ClientError(400, 'caption is a required field');
+  }
+  /**
+   * - create a url for the image by combining '/images' with req.file.filename
+   * - insert the "caption" and "url" into the "images" table
+   * - respond with the inserted row data
+   * - catch any errors
+   */
+  const url = req.file.filename + '/images';
```

Comment:
Close but this is backwards, the output should be something like: `/images/my-file-name.png`